--- xrdp-0.10.4.1/sesman/sesman.ini.in~	2025-07-07 12:37:24.000000000 +0000
+++ xrdp-0.10.4.1/sesman/sesman.ini.in	2026-01-02 11:35:35.637727491 +0000
@@ -12,13 +12,13 @@
 ReconnectScript=reconnectwm.sh
 
 [Security]
-AllowRootLogin=true
+AllowRootLogin=false
 MaxLoginRetry=4
-TerminalServerUsers=tsusers
-TerminalServerAdmins=tsadmins
+TerminalServerUsers=xrdp
+TerminalServerAdmins=root
 ; When AlwaysGroupCheck=false access will be permitted
 ; if the group TerminalServerUsers is not defined.
-AlwaysGroupCheck=false
+AlwaysGroupCheck=true
 ; When RestrictOutboundClipboard=all clipboard from the
 ; server is not pushed to the client.
 ; In addition, you can control text/file/image transfer restrictions
@@ -112,10 +112,10 @@
 
 [Logging]
 ; Note: Log levels can be any of: core, error, warning, info, debug, or trace
-LogFile=xrdp-sesman.log
+LogFile=/dev/null
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
@@ -150,8 +150,8 @@
 param=-noreset
 param=-nolisten
 param=tcp
-param=-logfile
-param=.xorgxrdp.%s.log
+#param=-logfile
+#param=.xorgxrdp.%s.log
 
 [Xvnc]
 param=Xvnc
--- xrdp-0.10.4.1/xrdp/xrdp.ini.in~	2025-07-07 12:37:24.000000000 +0000
+++ xrdp-0.10.4.1/xrdp/xrdp.ini.in	2026-01-02 11:41:07.518022458 +0000
@@ -4,7 +4,9 @@
 
 ; fork a new process for each incoming connection
 fork=true
-
+; IP address to listen
+; ;address=127.0.0.1
+;
 ; ports to listen on, number alone means listen on all interfaces
 ; 0.0.0.0 or :: if ipv6 is configured
 ; space between multiple occurrences
@@ -201,10 +203,10 @@
 
 [Logging]
 ; Note: Log levels can be any of: core, error, warning, info, debug, or trace
-LogFile=xrdp.log
+LogFile=/dev/null
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
@@ -240,17 +242,17 @@
 ; Some session types such as Xorg and Xvnc start a display server.
 ; Startup command-line parameters for the display server are configured
 ; in sesman.ini. See and configure also sesman.ini.
-[Xorg]
-name=Xorg
-lib=libxup.@lib_extension@
-username=ask
-password=ask
-port=-1
-code=20
+#[Xorg]
+#name=Xorg
+#lib=libxup.@lib_extension@
+#username=ask
+#password=ask
+#port=-1
+#code=20
 ; Frame capture interval (milliseconds)
-h264_frame_interval=16
-rfx_frame_interval=32
-normal_frame_interval=40
+#h264_frame_interval=16
+#rfx_frame_interval=32
+#normal_frame_interval=40
 
 [Xvnc]
 name=Xvnc
@@ -286,13 +288,13 @@
 ; Generic VNC Proxy
 ; Tailor this to specific hosts and VNC instances by specifying an ip
 ; and port and setting a suitable name.
-[vnc-any]
-name=vnc-any
-lib=libvnc.@lib_extension@
-ip=ask
-port=ask5900
-username=na
-password=ask
+#[vnc-any]
+#name=vnc-any
+#lib=libvnc.@lib_extension@
+#ip=ask
+#port=ask5900
+#username=na
+#password=ask
 #pamusername=asksame
 #pampassword=asksame
 #delay_ms=2000
@@ -309,15 +311,15 @@
 ; Generic RDP proxy using NeutrinoRDP
 ; Tailor this to specific hosts by specifying an ip and port and setting
 ; a suitable name.
-[neutrinordp-any]
-name=neutrinordp-any
+#[neutrinordp-any]
+#name=neutrinordp-any
 ; To use this section, you should build xrdp with configure option
 ; --enable-neutrinordp.
-lib=libxrdpneutrinordp.@lib_extension@
-ip=ask
-port=ask3389
-username=ask
-password=ask
+#lib=libxrdpneutrinordp.@lib_extension@
+#ip=ask
+#port=ask3389
+#username=ask
+#password=ask
 ; Uncomment the following lines to enable PAM authentication for proxy
 ; connections.
 #pamusername=ask
