--- xrdp-0.9.18.1/configure.ac~	2022-02-07 13:26:33.000000000 +0000
+++ xrdp-0.9.18.1/configure.ac	2022-02-22 17:53:14.649764980 +0000
@@ -182,9 +182,6 @@
 AX_CFLAGS_WARN_ALL
 AX_APPEND_COMPILE_FLAGS([-Wwrite-strings])
 
-AM_COND_IF([LINUX],
-  [AX_APPEND_COMPILE_FLAGS([-Werror])]) # bsd has warnings that have not been fixed yet
-
 AM_COND_IF([DEVEL_DEBUG],
   [AX_APPEND_COMPILE_FLAGS([-g -O0])],
   [AX_APPEND_COMPILE_FLAGS([-O2])])
