diff -Naur xrdp-0.4.1.orig/sesman/session.c xrdp-0.4.1/sesman/session.c
--- xrdp-0.4.1.orig/sesman/session.c	2007-05-28 01:30:34.000000000 +0200
+++ xrdp-0.4.1/sesman/session.c	2008-09-10 15:01:42.000000000 +0200
@@ -237,7 +237,7 @@
   }
   else if (pid == 0) /* child sesman */
   {
-    g_unset_signals();
+    /* g_unset_signals(); */
     auth_start_session(data, display);
     g_sprintf(geometry, "%dx%d", width, height);
     g_sprintf(depth, "%d", bpp);
